### ISSUE TITLE
Date Scalar type conflict

### ISSUE THREAD

**Issue by RuedigerLudwig** [nestjs/graphql#254]
## I'm submitting a...
[ ] Regression 
[ ] Bug report
[ ] Feature request
[X] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current & Expected behavior
I tried to build a scalar as described in the official documentations schema first. I expected, that I could use the generated interfaces to implement them in my actually used models. My models usually implement two interfaces one from the generated gql and one that enforces my database model.

In the generated types I would have expeced Date to be a Date type, but at the bottom is a
```typescript
type Date = any
```

This makes it impossible to extend my Model with both interfaces. I get an error:
<pre>
Interface 'Obj' cannot simultaneously extend types 'IdObject' and 'ObjGql'.
  Named property 'createdAt' of types 'IdObject' and 'ObjGql' are not identical.
</pre>

## Minimal reproduction of the problem with instructions
I did not include the files for scalar from the official documentation, but they are there as well of course.

```grapql
scalar Date

type ObjGql {
   id: ID!
   createdAt: Date!
   data: String!
}
```

```typescript
export interface ObjGql {
    id: string;
    createdAt: Date;
    data: string;
}
type Date = any
```

```typescript
import { ObjGql } from 'graphql.schema'

interface IdObject {
  id: string;
  createdAt: Date
}

interface Obj extends IdObject, ObjGql {}
```

## What is the motivation / use case for changing the behavior?
This all works perfectly once I remove the line
```typescript
type Date = any
```
from graphql.schema.ts

Is there any way to let the generating script know that the result will not be of type any, but of type Date?

Thank you for any hints or maybe updates in the documentation.

## Environment

<pre><code>
Nest version: 6.5.0
 
For Tooling issues:
- Node version: 10.15.3
- Platform: Ubuntu
</code></pre>

**Comment by kamilmysliwiec** [comment]
Please, provide a minimal repository which reproduces your issue. :)

**Comment by kamilmysliwiec** [comment]
Fixed in 6.3.1 :)
